config/ambd/toolchain/BUILD.gn

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+import("${build_root}/toolchain/gcc_toolchain.gni")
+
+declare_args() {
+  ambd_ar = ""
+  ambd_cc = ""
+  ambd_cxx = ""
+  ambd_cpu = ""
+}
+
+gcc_toolchain("ambd") {
+  ar = ambd_ar
+  cc = ambd_cc
+  cxx = ambd_cxx
+
+  toolchain_args = {
+    current_os = "freertos"
+    current_cpu = ambd_cpu
+    is_clang = false
+    import("${chip_root}/src/platform/AMBD/args.gni")
```

Comment:
same question here: could name be AmebaD for the platform?

We also seem to be inconsistent in casing currently: uppercase (ESP32, EFR32, KW32), somewhatcamel (Linux, Darwin, FreeRTOS, Zephyr) and lowercase (nrfconnect, qpg, telink).

Not sure what to do about it, but instinct would be to try to go for lowercase or camel. all-upper seems odd  to me.

Reply:
Thanks for the feedback,
The platform name will be changed to the camel style as 'Ameba' instead.